astrix-integration-tests/src/test/java/com/avanza/astrix/integration/tests/SpaceToServiceRegistryIntegrationTest.java

```diff
@@ -0,0 +1,88 @@
+import com.avanza.astrix.beans.registry.ServiceRegistryClient;
+import com.avanza.astrix.beans.service.ServiceProperties;
+import com.avanza.astrix.config.DynamicConfig;
+import com.avanza.astrix.config.GlobalConfigSourceRegistry;
+import com.avanza.astrix.config.MapConfigSource;
+import com.avanza.astrix.context.AstrixConfigurer;
+import com.avanza.astrix.context.AstrixContext;
+import com.avanza.astrix.gs.test.util.PuConfigurers;
+import com.avanza.astrix.gs.test.util.RunningPu;
+import com.avanza.astrix.integration.tests.domain.api.LunchService;
+import com.avanza.astrix.provider.component.AstrixServiceComponentNames;
+import com.avanza.astrix.test.util.AutoCloseableRule;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Ignore
+public class SpaceToServiceRegistryIntegrationTest {
+
+    @ClassRule
+    public static RunningPu serviceRegistrypu = PuConfigurers.partitionedPu("classpath:/META-INF/spring/service-registry-pu.xml")
+            .numberOfPrimaries(1)
+            .numberOfBackups(0)
+            .startAsync(false)
+            .configure();
+
+    private static MapConfigSource clientConfig = new MapConfigSource() {{
+        set(AstrixSettings.SERVICE_REGISTRY_URI, AstrixServiceComponentNames.GS_REMOTING + ":jini://*/*/service-registry-space?groups=" + serviceRegistrypu.getLookupGroupName());
+    }};
+
+    private static MapConfigSource config = new MapConfigSource() {{
+        set(AstrixSettings.SERVICE_REGISTRY_URI, AstrixServiceComponentNames.GS_REMOTING + ":jini://*/*/service-registry-space?groups=" + serviceRegistrypu.getLookupGroupName());
+        set(AstrixSettings.SERVICE_REGISTRY_EXPORT_RETRY_INTERVAL, 250);
+        set(AstrixSettings.BEAN_BIND_ATTEMPT_INTERVAL, 100);
+    }};
+
+    @ClassRule
+    public static RunningPu lunchPu = PuConfigurers.partitionedPu("classpath:/META-INF/spring/lunch-pu.xml")
+            .numberOfPrimaries(1)
+            .numberOfBackups(0)
+            .contextProperty("configSourceId", GlobalConfigSourceRegistry.register(config))
+            .startAsync(false)
+            .configure();
+
+    @Rule
+    public AutoCloseableRule autoCloseableRule = new AutoCloseableRule();
+
+    private AstrixContext clientContext;
+
+    @Before
+    public void setup() throws Exception {
+        this.clientContext = autoCloseableRule.add(new AstrixConfigurer().setConfig(DynamicConfig.create(clientConfig)).configure());
+    }
+
+    @Test
+    public void name() throws InterruptedException {
+        ServiceRegistryClient serviceRegistryClient = clientContext.getBean(ServiceRegistryClient.class);
+        List<ServiceProperties> providers = serviceRegistryClient.list(AstrixBeanKey.create(LunchService.class));
+
+        System.out.println(providers);
+
+        Thread.sleep(TimeUnit.MINUTES.toMillis(2));
```

Comment:
Two minutes is a long time :) (I saw that the test is @Ignored, so its fine but do we need it if its ignored?)